package.json

```diff
@@ -74,11 +78,10 @@
     "postcss-remove-rules": "^1.0.0",
     "postcss-selector-replace": "^1.0.2",
     "prismjs": "^1.29.0",
-    "rollup": "^3.7.5",
+    "rollup": "^2.70.1",
```

Comment:
Why is this a change to an earlier version?

Reply:
There is an issue running the build with the older versions of rollup.